src/devices/Media/VideoDevice/samples/Program.cs

```diff
@@ -20,7 +20,14 @@
 // Get the resolutions of the format
 foreach (var resolution in device.GetPixelFormatResolutions(PixelFormat.YUYV))
 {
-    Console.Write($"{resolution.Width}x{resolution.Height} ");
+    if (resolution is DiscreteResolution discreteRes)
+    {
+        Console.Write($"{discreteRes.Width}x{discreteRes.Height} ");
+    }
+    else if (resolution is StepwiseResolution stepRes)
+    {
+        Console.Write($"[{stepRes.MinWidth}x{stepRes.MinHeight}]->[{stepRes.MaxWidth}x{stepRes.MaxHeight}], Step [{stepRes.StepWidth},{stepRes.StepHeight}] ");
+    }
```

Comment:
should you handle continuous resolution here?

Reply:
In the UnixVideoDevice, I was handling the continuous as as stepwise with step 1